scss/5-pages/_report.scss

```diff
@@ -0,0 +1,80 @@
+
+.site-header {
+	color:#4d4d4d;
+	margin-top: 5rem;
+	margin-bottom: 5rem;
+
+    h1 {
+        font-size: 60pt;
+        margin-top: 60pt;
+        font-weight: bold;
+    }
+
+    h2{
+    	margin-top:0px;
+    }
+
+    margin-bottom: 2em;
+}
+
+
+.container.report {
+	color:#4d4d4d;
+	margin-bottom: 45pt;
```

Comment:
If you want to use px, then use the rem mixin, otherwise use rem :)

@jacmoe I have seen somewhere in the code pt it was used, so I sticked with it (thought it was scss specific, was not aware of this). I will fix it today, when I get back from work, I am not pro in css, so it's good to know where I am making mistakes.